### PR TITLE
Use different command to convert images with ImageMagick

### PR DESCRIPTION
**Pull request by ethanliu24**
## Proposed Changes

_(Describe your changes here. Also describe the motivation for your changes: what problem do they solve, or how do they improve the application or codebase? If this pull request fixes an open issue, [use a keyword to link this pull request to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).)_

In recent versions of ImageMagick, the `convert` command is deprecated, and it is recommended to use the `magick` command. This PR uses the recommended command to convert graphs to display in the export modal.
...

<details>
<summary>Screenshots of your changes (if applicable)</summary>

</details>

## Type of Change

_(Write an `X` or a brief description next to the type or types that best describe your changes.)_

| Type                                                                                    | Applies? |
| --------------------------------------------------------------------------------------- | -------- |
| 🚨 _Breaking change_ (fix or feature that would cause existing functionality to change) |          |
| ✨ _New feature_ (non-breaking change that adds functionality)                          |          |
| 🐛 _Bug fix_ (non-breaking change that fixes an issue)                                  |          |
| 🎨 _User interface change_ (change to user interface; provide screenshots)              |          |
| ♻️ _Refactoring_ (internal change to codebase, without changing functionality)          |          |
| 🚦 _Test update_ (change that _only_ adds or modifies tests)                            |          |
| 📦 _Dependency update_ (change that updates a dependency)                               |          |
| 🔧 _Internal_ (change that _only_ affects developers or continuous integration)         |X         |

## Checklist

_(Complete each of the following items for your pull request. Indicate that you have completed an item by changing the `[ ]` into a `[x]` in the raw text, or by clicking on the checkbox in the rendered description on GitHub.)_

Before opening your pull request:

- [X] I have performed a self-review of my changes.
  - Check that all changed files included in this pull request are intentional changes.
  - Check that all changes are relevant to the purpose of this pull request, as described above.
- [ ] I have added tests for my changes, if applicable.
  - This is **required** for all bug fixes and new features.
- [ ] I have updated the project documentation, if applicable.
  - This is **required** for new features.
- [ ] If this is my first contribution, I have added myself to the list of contributors.
- [X] I have updated the project Changelog (this is required for all changes).

After opening your pull request:

- [x] I have verified that the CircleCI checks have passed.
- [x] I have [requested a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from a project maintainer.

## Questions and Comments

_(Include any questions or comments you have regarding your changes.)_

### CODE REVIEW

**Review by david-yz-liu — APPROVED** [review]
@ethanliu24 great work, and in particular the research for fixing the `text-anchor` value.

**Review by david-yz-liu — COMMENTED** [review]
@ethanliu24 oops, there is one thing I forgot to flag for you. Currently there's a check on the server code that the `convert` executable is installed. This should also be changed to `magick`. Look at the Haskell setup files for the place I'm referring to.

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,8 +17,10 @@ jobs:
       - run:
           name: Install Package Dependencies
           command: |
-            sudo apt-get update
```
[the remark this answers is not in the document]
Comment (david-yz-liu) [reply .circleci/config.yml:20]: I meant the existing command that installed `texlive-latex-base` (just running `sudo apt-get install` without additional arguments doesn't do much)

[the remark this answers is not in the document]
Comment (ethanliu24) [reply .circleci/config.yml:20]: I'm not too familiar with linux (yet), but is there a reason for installing latex first before imagemagick?

**Pull request merged**